docs/creating_integration_file_structure.md

```diff
@@ -21,6 +21,10 @@ If your integration is going to integrate one or more devices, you will need to
 
 If your integration is going to register services, it will need to provide a description of the available services. The description is stored in `services.yaml`. [More information about `services.yaml`.](dev_101_services.md)
 
+## Data update coordinator - `coordinator.py`
+
+If you want your integration to fetch data, and you want to use a `DataUpdateCoordinator`, you need to create it in `coordinator.py`. [More information about `DataUpdateCoordinator`](integration_fetching_data.md/#coordinated-single-api-poll-for-data-for-all-entities).
```

Comment:
The guidance in `integration_fetching_data` is a bit more nuanced than this. There are multiple ways to fetch data including having the entity poll or having a coordinated polling when data is shared across entities.

Maybe something like:
```There are multiple ways for your integration to receive data including push or poll. Commonly integrations will fetch data with a single coordinated poll across all entities which requires using a `DataUpdateCoordinator`...```

Reply:
Thanks for this, this is definitely better 👍